src/osd/SnapMapper.h

```diff
@@ -281,6 +281,24 @@ class SnapMapper : public Scrub::SnapMapReaderI {
   tl::expected<object_snaps, SnapMapReaderI::result_t> get_snaps_common(
     const hobject_t &hoid) const;
 
+  /// file @out vector with the first objects with @snap as a snap
+  void get_objects_by_prefixes(
+    snapid_t snap,
+    unsigned max,
+    std::vector<hobject_t> *out);
+
+  std::set<std::string>           prefixes;
```

Comment:
a doubt: vertical alignment. In the coding standards I found very little in the matter:

```
// Assignment operators always have spaces around them.
x = 0;
```
> https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace

This doesn't prohibit using more spaces IIUC but I would advice honoring the already established convection per [`Be consistent with existing code`](https://google.github.io/styleguide/cppguide.html#Goals). For other members of this class we avoided vertical alignment.